.github/workflows/main.yml

```diff
@@ -91,22 +91,26 @@ jobs:
     - name: Configure CMake
       if: "matrix.platform.cmake"
       run: |
-        cmake -B build \
+        set -- \
           -DBUILD_SHARED_LIBS=ON \
           -DCMAKE_BUILD_TYPE=Release \
           -DSDL2IMAGE_DEPS_SHARED=OFF \
           -DSDL2IMAGE_SAMPLES=ON \
-          -DSDL2IMAGE_TESTS=ON \
-          -DSDL2IMAGE_TESTS_INSTALL=ON \
```

Comment:
You can leave `-DSDL2IMAGE_TESTS_INSTALL=ON` here.
It will automatically be disabled when either `SDL2IMAGE_TESTS` or `SDL2IMAGE_INSTALL` are disabled (because of `cmake_dependent_option`).

Reply:
Good thinking.

I'll come back to this when I have more time to investigate what's broken in the mostly-static CMake build.

Reply:
I followed this suggestion in #258.